doc/api-example.md

```diff
@@ -57,9 +57,221 @@ We can add sequences to the MinHash objects and query at any time --
 
 ```
 
+## Introduction: k-mers, molecule types, and hashing.
+
+### DNA k-mers
+
+The basis of sourmash is k-mers. Suppose we have a 35 base DNA sequence, and
+we break it into four 31-mers:
+```
+>>> K = 31
+>>> dnaseq = "ATGCGAGTGTTGAAGTTCGGCGGTACATCAGTGGC"
+>>> for i in range(0, len(dnaseq) - K + 1):
+...    kmer = dnaseq[i:i+K]
+...    print(i, kmer)
+0 ATGCGAGTGTTGAAGTTCGGCGGTACATCAG
+1 TGCGAGTGTTGAAGTTCGGCGGTACATCAGT
+2 GCGAGTGTTGAAGTTCGGCGGTACATCAGTG
+3 CGAGTGTTGAAGTTCGGCGGTACATCAGTGG
+4 GAGTGTTGAAGTTCGGCGGTACATCAGTGGC
+
+```
+
+sourmash provides a hash function (by default MurmurHash) that will convert
+each k-mer into 64-bit numbers. The easiest way to access this hash
+function is via the `seq_to_hashes` method on `MinHash` objects, which
+returns a list:
+```
+>>> from sourmash import MinHash
+>>> mh = MinHash(n=50, ksize=K)
```

Comment:
Is there a reason you're using `num` here? I think if we want to encourage folks to to use `scaled` as a default, or even think about `scaled`, it would be better to use `scaled=1` here?

Reply:
I agree, it's literally just so that we can get a `MinHash` object. And I didn't want to complicate the object creation API call. 🤷 I can fix, tho, since you flagged it 😁

Reply:
fixed in a79a331584